front/app/containers/App/UserSessionRecordingModal/index.tsx

```diff
@@ -66,7 +68,15 @@ const UserSessionRecordingModal = () => {
         </Box>
 
         <Text fontSize="l">{formatMessage(messages.modalDescription1)}</Text>
-        <Text fontSize="l">{formatMessage(messages.modalDescription2)}</Text>
+        <Text fontSize="l">
+          {formatMessage(messages.modalDescription2)}{' '}
+          <Link
+            to="https://docs.google.com/document/d/1a1hlDcMHIJrSJb2sD1AenOGfWpjxcsrzkjrDhlTtzNw"
```

Comment:
It was agreed that it was fine to link directly to this Google doc - only turned on for US customers so English only isn't a problem